### PR TITLE
Process Data Function Bug Fix

### PR DESCRIPTION
**Pull request by Biebras**
The current function produces an error if the function tries to process data with a room that does not exist. This can happen for a few seconds when the host disconnects and a client sends a message to the room that does not exist.

### CODE REVIEW

**Comment by Biebras** [comment]
Waiting for approval.

**Review by Speidy674 — APPROVED** [review]

**Pull request merged**
